I/IGLWrap/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
Do not put files in the same directory as `build_tarballs.jl` so that you have to do
```julia
DirectorySource(".")
```
That's known to be buggy at the moment.  Please move to a subdirectory (e.g. `bundled/`, like all other packages do)